commit 7c00ad212472113ce62348408a0710146f8afefc
Author: Sage Weil <sweil@redhat.com>
Date:   Wed Aug 8 08:51:10 2018 -0500

    mgr/MgrClient: make some noise for a user if no mgr daemon is running
    
    Otherwise a cli command may unexpectedly hang forever with not output
    because no ceph-mgr is running, which can be confusing for an admin.
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index 06d2810d827..ea6998167b7 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -428,7 +428,7 @@ int MgrClient::start_command(const vector<string>& cmd, const bufferlist& inbl,
     MCommand *m = op.get_message({});
     session->con->send_message(m);
   } else {
-    ldout(cct, 4) << "start_command: no mgr session, waiting" << dendl;
+    ldout(cct, 0) << "no mgr session (no running mgr daemon?), waiting" << dendl;
   }
   return 0;
 }
